inspector/transform.py

```diff
@@ -16,12 +19,15 @@ def raw(meta, task, task_dir, stdout, stderr) -> list[str]:
 
 def fetch_geekbench_results(meta, task, task_dir, stdout, stderr) -> list[str]:
     outputs: list[str] = []
-    urls = re.findall(re.compile(r'https://[^\s"]*/v6/cpu[^\s"]*'), stdout.decode("utf-8"))
+    urls = re.findall(
+        re.compile(r'https://[^\s"]*/v6/cpu[^\s"]*'), stdout.decode("utf-8")
+    )
     if urls:
         res = requests.get(urls[0])
         assert 200 <= res.status_code < 300, f"Status code is {res.status_code}"
-        with open(os.path.join(task_dir, "results.html"), "w") as f:
-            f.write(res.text)
-            outputs.append("results.html")
+        results = geekbench_html_to_json(res.content)
```

Comment:
This will return bytes, however the method -according to its type hints- expect string.
`res.text` will be string.